docs/pages/careers/design-engineer-x-grid.md

```diff
@@ -38,11 +38,17 @@ For additional details about the culture you can check our [careers](https://mui
 
 ## Why we're hiring
 
-Both our open-source community and our premium products are growing fast (x2 YoY).
-We need talented people to keep that going!
+[According to our last survey](https://mui.com/blog/2021-developer-survey-results/#what-are-your-most-important-criteria-for-choosing-a-ui-library), the quality of the design execution (UX & UI) is the most important criterion used by developers & designers to pick their next general-purpose UI library.
+
+We hear an even bigger pain around _design_ from developers who adopt projects specialized into one UI component, e.g. data grid. For them, it's not only the default behaviors that need to be great but also the ease of customizing the components to their needs and integrating into the rest of their application.
+
+Up until now, we solved a subset of these problems by specializing in a MUI X Data Grid that deeply integrates with Material UI.
+We feel that the timing is right to take a step further and invest in the design of MUI X to have the best data grid not only for Material UI but for **any** React project.
 
-We are aiming to make high-end design accessible to the many, empowering developers who lack access to a designer as well as saving time for designers with a source they can trust.
-[According to our last survey](https://mui.com/blog/2021-developer-survey-results/#what-are-your-most-important-criteria-for-choosing-a-ui-library), the quality of the design execution (UX & UI) is the most important criterion used by developers & designers to pick their next UI library.
+The MUI X Data Grid team has no dedicated designer, so we are opening this role to give us the means of this objective.
```

Comment:
```suggestion

```

I would remove this. My argument is that this sentence puts the person in a state of mind that they should be a designer, which we discussed that it's not the goal, but even if we keep it I'm unsure how much it will affect people's decision to apply (or not).

Reply:
+1

Reply:
+ 1, agreed! I think the rest of the paragraph already conveys the justification for the role, no need for "so we are opening this role...".